charts/ia-home-office-integration-api/values.yaml

```diff
@@ -5,6 +5,7 @@ java:
   environment:
     IDAM_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
     S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    HOME_OFFICE_ENDPOINT: "http://ia-home-office-mock-api-aat.service.core-compute-aat.internal"
```

Comment:
should be able to use `{{ .Values.global.environment }}` instead of `aat`

Reply:
Will change this